src/main/java/seedu/duke/command/DeleteCommand.java

```diff
@@ -30,24 +29,13 @@ public DeleteCommand(String input) {
      * @param input the input string from the user
      * @return the name of the secret to be deleted
      */
-    public String extractName(String input) {
```

Comment:
Since this is used so much, it should be put into the Command function. I will make the according changes